src/components/ApiDocumentation/__tests__/ApiTypeIndicator.test.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { ApiTypeIndicator } from '../ApiTypeIndicator';
+
+describe('ApiTypeIndicator', () => {
+  it('renders when not known', () => {
+    let wrapper = shallow(<ApiTypeIndicator apiType="unknown" />);
+    expect(wrapper.html()).not.toContain('img');
+  });
+
+  it('renders when known', () => {
```

Comment:
The test names do not explicit what is being under test.

Reply:
I will change it